### PR TITLE
DTSPO-6100 - Subscription targeting

### PR DESCRIPTION
**Pull request by endakelly**
This is what it would look like. It works with assigning multiple policies at the subscripton level. Currently using hardcoded values until I figure out how to properly variablise the values.
image

### CODE REVIEW

File: .github/workflows/manage-azure-policy.yml [hunk 1]
```diff
@@ -42,8 +42,42 @@ jobs:
...
+        creds: ${{secrets.AZURE_CREDENTIALS}}
+        allow-no-subscriptions: false
+
+    - name: Live - Create or Update Azure Policies
+      uses: azure/manage-azure-policy@v0
+      with:
+        mode: complete
+        paths: |
+          policies/**
+          assignments/subscriptions/${{env.POLICY_SUBSCRIPTION}}/**
```
Comment (timja) [inline .github/workflows/manage-azure-policy.yml:70]: can't you just apply to all sub folders ?

Comment (endakelly) [reply .github/workflows/manage-azure-policy.yml:70]: That will work if the matrix is removed and the compliance scan step has the scopes updated instead:
`        scopes: |
          /subscriptions/2307d175-7e49-434b-9ac2-515529b845f2`

That would leave just an action job for the subscription and the management group. 

Any preference?

Comment (timja) [reply .github/workflows/manage-azure-policy.yml:70]: do you need the compliance scan? 

the management group one already does all subscriptions under it, alternatively you can do the same just create a shell script to find all subscriptions required for the compliance scan

Comment (endakelly) [reply .github/workflows/manage-azure-policy.yml]: Makes sense. Updated as suggested

File: .github/workflows/manage-azure-policy.yml [hunk 2]
```diff
@@ -42,8 +42,34 @@ jobs:
...
         scopes: |
           /subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb
-  
-  apply-azure-policy-live:
+
+  apply-azure-policy-subscription:
+    if: ${{ github.ref == 'refs/heads/master' }}
+    strategy:
+      matrix:
+        subscriptions: [2307d175-7e49-434b-9ac2-515529b845f2]
```
Comment (timja) [inline .github/workflows/manage-azure-policy.yml:50]: I still don't understand why you need this

File: .github/workflows/manage-azure-policy.yml [hunk 3]
```diff
@@ -42,8 +42,34 @@ jobs:
...
+    runs-on: ubuntu-latest
+    env:
+      POLICY_SUBSCRIPTION: ${{ matrix.subscriptions }}
```
Comment (timja) [inline .github/workflows/manage-azure-policy.yml:53]: why do you need this

File: .github/workflows/manage-azure-policy.yml (hunk 2 above)
Comment (endakelly) [reply .github/workflows/manage-azure-policy.yml]: I had meant to remove that. Updated.

File: .github/workflows/manage-azure-policy.yml (hunk 3 above)
Comment (endakelly) [reply .github/workflows/manage-azure-policy.yml]: I had meant to remove that. Updated.

**Review by timja — APPROVED** [review]
looks right

**Pull request merged**